tree/30_generic_methods/file_copy_from_local_source.cf

```diff
@@ -30,8 +30,11 @@ bundle agent file_copy_from_local_source(source, destination)
 {
   vars:
      "class_prefix"     string => canonify(join("_", "this.callers_promisers"));
+      "args"             slist => { "${source}", "${destination}" };
 
   methods:
       "copy without recursion" usebundle => file_copy_from_local_source_recursion("${source}", "${destination}", "0");
       "new result classes"     usebundle => _classes_copy("${class_prefix}_copy_without_recursion", "${class_prefix}");
+
+      "report"                 usebundle => _logger("Copying ${destination} from ${source}", "", @{args});
```

Comment:
No old class to signify that only new style reporting should be done here.
Old style reporting will be done in the submethod as it did before.